fehbg.nu

```diff
@@ -20,13 +20,13 @@
 # xinitrc.
 #
 # Dependencies;
-# * nu version  >0.25.1 (0.25.1 doesn't work due to coercion error bug)
+# * nu version  >=0.31.1 (requires new syntax introduced in 0.31.1 and 0.32.0)
```

Comment:
Can you change this line to just `# * nu version  >=0.32.0 (requires new syntax introduced in 0.32.0)`?